kraken-std/src/kraken/std/helm/helmapi.py

```diff
@@ -23,7 +23,7 @@ def helm_package(
         raise ValueError("output_file and output_directory cannot both be set")
 
     with contextlib.ExitStack() as exit_stack:
-        command = ["helm", "package", str(chart_path)]
+        command = ["helm", "package", str(chart_path), "--dependency-update"]
```

Comment:
@callumPearce can we make this configurable with a keyword argument `dependency_update` in `helm_package`? (for which defaulting to `True` is reasonable IMO)

```suggestion
        command = ["helm", "package", str(chart_path)]
        if dependency_update:
            command.append("--dependency-update")
```

Reply:
Yes, definitely!

Reply:
https://github.com/kraken-build/kraken-build/pull/30/commits/8493560cd7bec86287cb1e82e0fc8dfaa6412015